lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -549,7 +552,53 @@ def upload_csv_to_report_store(rows, csv_name, course_id, timestamp):
     )
 
 
-def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input, action_name):
+def verification_status_for_user(user, course_id, user_enrollment_mode):
+    """
+    Returns the verification status.
+    """
+    if user_enrollment_mode not in CourseMode.VERIFIED_MODES:
+        return 'ID Verified'
```

Comment:
I am unsure about this. But i think `ID Verified` make sense for users who are not in `verified` and `professional` track